R/process_extract_delayed_discharges.R

```diff
@@ -0,0 +1,87 @@
+#' Process the delayed discharges extract
+#'
+#' @description This will read and process the
+#' delayed discharges extract, it will return the final data
+#' but also write this out as a zsav and rds.
+#'
+#' @param year The year to process, in FY format.
+#' @param data The extract to process
```

Comment:
```suggestion
#' @param data The extract to process
#' @param year The year to process, in FY format.
```

Just to match parameter shuffling below

Reply:
Good spot!